helm/net-exporter-chart/values.yaml

```diff
@@ -17,6 +17,10 @@ image:
   repository: giantswarm/net-exporter
   tag: [[ .SHA ]]
 
+namespaceLabeler:
+  image:
+    name: giantswarm/namespace-labeler
```

Comment:
Just FMI what is this?

Reply:
to ensure networkPolicy works, each namespace needs to be labeled ( in the most default case `name=nsName`) but thats not the case by default so this image check if namespace is labeled and if tis not it will add the label

Reply:
I just adding this values  to the values.yaml so is consistent in both `cert-operator` and `cert-operator-chart` directories